### ISSUE TITLE
geeting error while installing mediapipe

### ISSUE THREAD

**Issue by subh-sk** [google/mediapipe#4295]
### OS Platform and Distribution

windows 11

### Programming Language and version

python 3.11.3

### Android AAR (if android)

None

### Describe the problem

```shell
I have python 3.11.3 version but I can't install mediapipe it's shows me Could not find a version that satisfies the requirement mediapipe
```

### Complete Logs

```shell
help me to resolve this problem
```

**Comment by ayushgdev** [comment]
Hello @subh-sk 
Python 3.11 is not supported by Mediapipe yet. The last version upgrade happened in [v0.8.9](https://github.com/google/mediapipe/releases/tag/v0.8.9) document. Please use Python 3.7 to 3.10. Attaching the relevant thread https://github.com/google/mediapipe/issues/1325 for better understanding.

**Comment by subh-sk** [comment]
@ayushgdev yes it resolved when i used 3.10 version thankyou

**Comment by ayushgdev** [comment]
Thank you . We shall close this issue
